docs/index.md

```diff
@@ -2,16 +2,32 @@
 
 Pulse displays metrics that provide insights about the current and historic performance of your software delivery process. To calculate these metrics, Pulse must collect information from key events of your particular software development workflow.
 
-Currently, Pulse supports a push-based integration with your workflow and the recommended way of sending the necessary data to Pulse is by using the Pulse CLI.
+Currently, Pulse supports a push-based integration with your workflow and the recommended way of reporting the necessary events to Pulse is by using the Pulse CLI.
 
-However, in some scenarios it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are sending data from providers that only support webhooks. For these situations, you can call an HTTP POST webhook instead. For example:
+## Before you begin
 
-`https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>`
+Consider the following before integrating your workflow with Pulse:
 
-!!! important
-    **Please let us know if you are planning on pushing events to Pulse using the webhook** so we can give you more detailed instructions on how to use it.
+-   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are reporting events from providers that only support webhooks. For these situations, you can call an HTTP POST webhook instead. For example:
+
+    ```test
+    https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>
+    ```
 
-The next sections include detailed instructions on how to complete the integration setup process.
+    !!! important
+        **Please let us know if you are planning on reporting events to Pulse using the webhook** so we can give you more detailed instructions on how to use it.
+
+-   When reporting events to Pulse you can use the optional field `system` to associate each event with the corresponding CVS repository.
```

Comment:
We should consider renaming the field `system` to something more straightforward and meaningful. For example, as Rafael suggested, it could be called `repository`.

Reply:
We decided to keep the name "system": https://codacy.atlassian.net/browse/PU-139?focusedCommentId=44655